src/Package/ComparePackageRepo.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace SiteStudio\Package;
+
+use Symfony\Component\Yaml\Yaml;
+use SiteStudio\Package\ArrayDiff;
+use SiteStudio\Package\ConfigInitializer;
+
+class ComparePackageRepo extends ConfigInitializer {
+
+    /**
+     * ConfigInitializer constructor.
+     */
+    public function __construct() {
+        parent::__construct();
+        $this->yamlDiffReport();
+    }
+
+    /**
+     * Yaml Difference Report.
+     */
+    public function yamlDiffReport() {
```

Comment:
@rakshithtb recommend splitting this up into logical functions as rationale is this allows us to more easily unit tests different stages, provide mocks. We want to allow test coverage due to the role this plays in the development process.

e.g.

yamlDiffReport()
* Generating list of projects
* Download report
* Invoke comparison logic